tests/Domain/Composer/Version/DrupalCoreVersionResolverTest.php

```diff
@@ -446,8 +446,9 @@ public function testResolvePredefinedVersionNotFound($version): void {
     $resolver->resolvePredefined($version);
   }
 
-  public function providerResolvePredefinedVersionNotFound(): array {
-    $data = $this->providerVersions();
+  public static function providerResolvePredefinedVersionNotFound(): array {
+    $obj = new DrupalCoreVersionResolverTest('testResolvePredefinedVersionNotFound');
+    $data = $obj->providerVersions();
```

Comment:
I think you can still use the original function if you just call it statically:

```suggestion
    $data = self::providerVersions();
```